docs/cube/index.rst

```diff
@@ -0,0 +1,13 @@
+.. _cube:
+
+************************************
+Cube Style Analysis (`gammapy.cube`)
+************************************
+
+.. currentmodule:: gammapy.cube
+
```

Comment:
Maybe add a sentence to say what cube analysis is and how it's an alternative to classical analysis

https://github.com/gammapy/PyGamma15/blob/gh-pages/talks/analysis-cube/2015-11-16_PyGamma15_Eger_Cube_Analysis.pdf

https://github.com/gammapy/PyGamma15/blob/gh-pages/talks/analysis-classical/2015-11-16_PyGamma15_Terrier_Classical_Analysis.pdf